### PR TITLE
Checkpoint openid connect client

### PR DESCRIPTION
**Pull request by anjumfatima90**
#23708

### CODE REVIEW

File: dev/io.openliberty.checkpoint_fat_security_oidc_social/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.op/jvm.options [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_security_oidc_social/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.op/jvm.options:1]: Shouldn't this have:
```
-Dcom.ibm.ws.beta.edition=true
```

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
Instant on build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_HGDcgONnEe2xRt1mHVwGHw&tab=com.ibm.team.build.web.ui.internal.editors.result.test.TestProvider

**Comment by anjumfatima90** [comment]
#build

**Review by tjwatson — DISMISSED** [review]

File: dev/io.openliberty.checkpoint_fat_security_oidc_social/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.op/jvm.options (hunk 1 above)
Comment (anjumfatima90) [reply dev/io.openliberty.checkpoint_fat_security_oidc_social/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.op/jvm.options:1]: We are not doing checkpoint for this server. Thanks!

**Comment by anjumfatima90** [comment]
#libby

**Comment by anjumfatima90** [comment]
#libby

**Review by tjwatson — APPROVED** [review]

**Pull request merged**